nms/packages/magmalte/app/views/subscriber/SubscriberTable.js

```diff
@@ -199,76 +201,130 @@ const StyledMenu = withStyles({
   />
 ));
 
+// export type SubscriberActionType = 'add' | 'edit' | 'delete';
+
+// type SubscriberError = $Values<typeof SUBSCRIBER_ADD_ERRORS>;
+
 function SubscriberActionsMenu(props: {onClose: () => void}) {
   const [anchorEl, setAnchorEl] = React.useState(null);
   const [open, setOpen] = React.useState(false);
   const [error, setError] = React.useState('');
   const enqueueSnackbar = useEnqueueSnackbar();
   const ctx = useContext(SubscriberContext);
   const successCountRef = useRef(0);
+  const [
+    subscriberAction,
+    setSubscriberAction,
+  ] = useState<SubscriberActionType>('add');
   const handleClick = event => {
     setAnchorEl(event.currentTarget);
   };
 
   const handleClose = () => {
     setAnchorEl(null);
   };
+
   /**
-   * Post subscriber chunk or update subscriber(s) if param edit is set to true
+   * Delete array of subscribers (susbcribers from csv) or array of subscriber IMSI (selected subscribers).
    *
-   * @param {Array<SubscriberInfo>} subscribers Array of subscribers to Add or Edit.
-   * @param {boolean} edit Update existing subscriber(s) if set to true.
+   * @param {selectedSubscribers} selectedSubscribers Array of subscriber IMSI to delete
+   * @param {Array<SubscriberInfo>} subscribers Array of subscribers to delete
    */
-  const saveSubscribers = async (
+  const deleteSubscribers = async (
+    selectedSubscribers: Array<string>,
     subscribers: Array<SubscriberInfo>,
-    edit: boolean,
   ) => {
```

Comment:
Change the function signature to something simpler:
```
const deleteSubscribers = async (subscribers: Array<string>) => {
```

The caller should be responsible for taking `Array<SubscriberInfo>`, and mapping that into a `Array<string>` to pass into this function